src/screens/Wallets/components/Header.tsx

```diff
@@ -93,17 +92,17 @@ export function HomeHeader(): JSX.Element {
         <Spacer horizontal value={scale(20)} />
         <Button onPress={navigateToNotifications}>
           <NotificationIcon color="#393b40" />
-          {unreadNotificationCount + newNotificationsCount > 0 && (
+          {newNotificationsCount > 0 && (
             <View style={styles.notificationCountContainer}>
               <Text color="white" fontSize={11} fontFamily="Inter_600SemiBold">
-                {unreadNotificationCount + newNotificationsCount}
```

Comment:
do we need this **unreadNotificationCount**, because newNotificationsCount will return all notif?

Reply:
if it returns all notif, then you are right, we do not need `unreadNotificationCount`.